queries/swift/highlights.scm

```diff
@@ -3,7 +3,6 @@
 
 ; Identifiers
 (attribute) @variable
-(simple_identifier) @variable
```

Comment:
Any reason to remove this?

Reply:
To reflect upstream changes; I believe it's replaced by 
```
(pattern bound_identifier: (simple_identifier)) @variable
```
further down.

Reply:
In https://github.com/alex-pinkus/tree-sitter-swift/pull/135, the PR author complained that it was “too broad, it overlaps with many other patterns.” I’m not sure why that was causing a problem but it seemed fine to me to narrow it.